Makefile

```diff
@@ -0,0 +1,24 @@
+# Docker image configuration
+IMAGE_NAME := rafayak/ovpn-connector
+IMAGE_TAG ?= latest
+
+# Builds a given docker container image.
+.PHONY: $(IMAGE_NAME)
+$(IMAGE_NAME):
+	@echo "Building Docker image $@:$(IMAGE_TAG)..."
+	docker build -f Dockerfile -t $@:$(IMAGE_TAG) .
+
+## image:			Builds docker image
+.PHONY: image
```

Comment:
The `Makefile` here just provides a simple way to build and push the container, allowing users to do:

```bash
make image push
```

to build and push the Docker image.